core/cli/src/plugin/merge-commands.ts

```diff
@@ -2,10 +2,24 @@ import type { CommandTask, Plugin } from '@dotcom-tool-kit/plugin'
 import type { ValidPluginsConfig } from '@dotcom-tool-kit/config'
 import { Conflict, isConflict } from '@dotcom-tool-kit/conflict'
 import { isDescendent } from './is-descendent'
+import { Logger } from 'winston'
+import { styles as s } from '@dotcom-tool-kit/logger'
+import path from 'path'
 
-export const mergeCommands = (config: ValidPluginsConfig, plugin: Plugin) => {
+export const mergeCommands = (config: ValidPluginsConfig, plugin: Plugin, logger: Logger) => {
   if (plugin.rcFile) {
-    for (const [id, configCommandTask] of Object.entries(plugin.rcFile.commands)) {
+    let commands = plugin.rcFile.commands
+
+    if (plugin.rcFile.hooks) {
```

Comment:
**nitpick**: could you add a TODO or HACK comment here so that it's easier to remember to delete this in a future major version?